Documentation/configuration/sbom.rst

```diff
@@ -31,22 +31,25 @@ Prerequisites
 Download SBOM
 =============
 
-The SBOM can be downloaded from the supplied Cilium image using the
-``cosign download sbom`` command.
+You can download the SBOM in-toto attestation from the supplied Cilium image using the following command:
 
 .. code-block:: shell-session
 
-    $ cosign download sbom --output-file sbom.spdx <Image URL>
+    $ cosign download attestation --predicate-type spdxjson --output-file cilium_SBOM.spdx.json <Image URI>
 
-Verify SBOM Image Signature
-===========================
+Verify SBOM attestation
+=======================
 
-To ensure the SBOM is tamper-proof, its signature can be verified using the
-``cosign verify`` command.
+To verify the SBOM in-toto attestation on the supplied Cilium image, run the following command:
 
 .. code-block:: shell-session
 
-    $ COSIGN_EXPERIMENTAL=1 cosign verify --certificate-github-workflow-repository cilium/cilium --certificate-oidc-issuer https://token.actions.githubusercontent.com --attachment sbom <Image URL> | jq
+    $ cosign verify-attestation --certificate-github-workflow-repository cilium/cilium --certificate-oidc-issuer https://token.actions.githubusercontent.com --type spdxjson <Image URI> | jq
```

Comment:
I think it would be useful to precise what the user should expect from the output (is it still the `Issuer` and `Subject` fields that the user should look at?).

Looks OK from my side otherwise, thanks! Although I'm still unfamiliar with cosign, and will rely on André to validate the changes.

Reply:
> I think it would be useful to precise what the user should expect from the output (is it still the Issuer and Subject fields that the user should look at?).

I agree (yes).